test/vars/CloudSpec.groovy

```diff
@@ -291,4 +291,19 @@ class CloudSpec extends JenkinsPipelineSpecification {
         0 * getPipelineMock("sh")("podman rmi -f hello || date")
         0 * getPipelineMock("sh")("podman rmi -f  || date")
     }
-}
+
+    /////////////////////////////////////////////////////////////////////
+    // updateQuayImageDescription
+    
+    def "[cloud.groovy] updateQuayImageDescription simple run"() {
+        setup:
+        groovyScript.getBinding().setVariable('QUAY_TOKEN', 'quaytoken')
+        when:
+        groovyScript.updateQuayImageDescription('DESCRIPTION', 'namespace', 'repository', [token: 'TOKEN'])
```

Comment:
to define additional cases for 

- ` [token: X, usernamePassword: Y]`
- ` [usernamePassword: Y]`
- ` []`